index.html

```diff
@@ -0,0 +1,288 @@
+						</div>
+				</label>
+				<label for="pro">
+					<input hidden type="radio" name="radio2" id="pro" value="pro">
+						<div class="select-tone select-tone2">
+							<p>Corporate / Pro</p>
+						</div>
+				</label>
+				<label for="media">
+					<input hidden type="radio" name="radio2" id="media" value="media">
+						<div class="select-tone select-tone3">
+							<p>Media / Formal</p>
+						</div>	
+				</label>
+			</div>
+		</div>
+	</div>
+	<div class="four">
+		<div class="upload2">
+			<p class="cirkle2">4</p>
+			<h3>Tells us how urgent this job is</h3>
+		</div>
+		<input type="range" min="0" max="100" step="50">
+		<div class="timeline">
+			<p>I got time</p>
+			<p>Average</p>
+			<p>Yesterday</p>
+		</div>
+		<div class="info">
+			<div class="timeline_hdr"><p>Timeline info</p></div>
+			<div class="timeline_disc"><p>With an <span>average</span> timeline set up, our expert translator can take a reasonable amount of time perfecting your translation</p></div>
+		</div>
+	</div>
+	</section>
+	<section class="five">
+		<div class="upload">
+			<p class="cirkle2">5</p>
+			<h3>Sing up and choose a payment method</h3>
+		</div>
+		<div class="have_acc">
+			<p>Already have an account with us? <a href="#">Sing in</a>.</p>
+		</div>
+		<div class="sign_form">
+			<div class="f_name">
+			<p>Full name</p>
+			<input type="text" name="name" maxlength="40" class="" required="">
+			<p>Email adress</p>
+			<input type="text" name="name" maxlength="40" class="" required="">
+			</div>
+			<div class="pass">
+			<p>Choose a password</p>
+			<input type="password" name="name" maxlength="40" class="" required="">
+			</div>
+		</div>
+		<div class="payment">
+				<label for="card">
+					<input hidden type="radio" name="radio3" id="card" value="card">
+						<div class="select-payment">
+							<img src="css/img/card.png">
+						</div>
+				</label>
+				<label for="paypal">
+					<input hidden type="radio" name="radio3" id="paypal" value="paypal">
+						<div class="select-payment">
+							<img src="css/img/paypal.png">
+						</div>
+				</label>
+				<label for="amazon">
+					<input hidden type="radio" name="radio3" id="amazon" value="amazon">
+						<div class="select-payment">
+							<img src="css/img/amazon.png">
+						</div>	
+				</label>
+		</div>
+		<div class="sign_form card_form">
+			<div class="f_name card_number">
+			<p>Card number</p>
+			<input type="text" name="name" maxlength="16" class="" required="">
+			</div>
+			<div class="f_name ex_date">
+			<p>Expiry date</p>
+			<input type="text" name="name" maxlength="8" class="" required="" placeholder="DD / MM / YY">
+			</div>
+			<div class="f_name code">
+			<p>code</p>
+			<input type="password" name="name" maxlength="3" class="" required="">
+			</div>
+		</div>
+		<div class="last_step">
+			<input id="checkBox" type="checkbox">
+			<p>I agree to the <a href="#">Terms &#38; Condiions</a></p>
+			<button class="act_btn">Place order</button>
```

Comment:
`button` element without explicit type defaults to `submit` type. which means that every button will attempt to send a form to the server.